src/main/java/internet/shop/security/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package internet.shop.security.impl;
+
+import internet.shop.exceptions.AuthenticationException;
+import internet.shop.lib.Inject;
+import internet.shop.lib.Service;
+import internet.shop.model.User;
+import internet.shop.security.AuthenticationService;
+import internet.shop.service.UserService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject private UserService userService;
```

Comment:
```suggestion
    @Inject 
    private UserService userService;
```
for me in this way it's more readable especially that in other services you put annotation above fields